lib/config/logging.go

```diff
@@ -91,6 +91,7 @@ func (l *LokiConfig) Validate() error {
 
 type GrafanaConfig struct {
 	BaseUrl      *string `toml:"base_url"`
+	BaseUrlCI    *string `toml:"base_url_github_ci"` // Base URL for the dashboard via GAP proxy used on CI
```

Comment:
`// Base URL for the dashboard via GAP proxy used on CI` -> `// URL of GAP proxy used on CI for Grafana`